src/FluentValidation/ValidatorMetadata.cs

```diff
@@ -74,11 +74,16 @@ public class PropertyValidatorOptions {
 		/// </summary>
 		public Func<PropertyValidatorContext, object> CustomStateProvider { get; set; }
 
+		/// <summary>
+		/// Function used to retrieve the severity for the validator
+		/// </summary>
+		public Func<PropertyValidatorContext, Severity> CustomSeverityProvider { get; set; }
+
 		/// <summary>
 		/// Severity of error.
 		/// </summary>
 		public Severity Severity { get; set; }

```

Comment:
Let's get rid of this entirely (see explanation in the review summary)

Reply:
Will do so.